syntax = "proto3";
package simproto;

service SimControl {
    rpc ControlSim(SimCommand) returns (Ack) {}
}

message SimCommand {
    enum SimCommandEnum {
        RunSimulator = 0;
        PauseSimulator = 1;
        StopSimulator = 2;
    }
    SimCommandEnum command = 1;
}

message Ack {
    bool success = 1;
	string message = 2;
}

